packages/iconset/stories/iconsList.json

```diff
@@ -892,7 +892,21 @@
         "workflowadd",
         "wrench",
         "zoomin",
-        "zoomout"
```

Comment:
Much of this data could be sourced from the `@adobe/spectrum-css-workflow-icons/dist/icons.json` in a future iteration (would need a little clean-up because it's pascal case with *.svg postfixes.